docs/Contributing/Release.md

```diff
@@ -0,0 +1,143 @@
+# Release Process
+
+
+## Prerequisites:
+- Prepare release notes
+
+
+## Weekly chores
+### Enarx dependency update
+- Update local code
+  ```bash
+  git fetch --all
+  git checkout main
+  git pull --ff-only
```

Comment:
Why not just 
```suggestion
  git fetch --all
  git checkout origin/main
```
?

This would work for any setup, assuming that enarx remote is named `origin`